pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -147,13 +148,27 @@ pub mod pallet {
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config> {
-		Todo,
+		/// A bonded token pool has been moved to destroying state. [pool_id]
+		DestructionStarted(T::PoolId),
+		/// Collateral distribution to bonded token holders has been completed for this pool - no more tokens or no more collateral to distribute. [pool_id]   
+		RefundCompleted(T::PoolId),
+		/// A bonded token pool has been fully destroyed and all collateral and deposits have been refunded. [pool_id]
+		Destroyed(T::PoolId),
 	}
 
 	// Errors inform users that something went wrong.
 	#[pallet::error]
 	pub enum Error<T> {
-		Todo,
+		/// The pool id is not currently registered.
+		PoolUnknown,
+		/// The amount to mint, burn, or swap is zero.
+		ZeroAmount,
+		/// The pool is already in the process of being destroyed.
+		Destroying,
```

Comment:
This would have to work as well as an error for when you are trying to mint or burn but the pool is in destroying state, so I'd strike the 'already' also in the description. If you think that's better, we could go for a more general `NotLive` error?

Reply:
Yeah something like `NotLive`, or `NotActive` would work as well.

Reply:
It's called `PoolNotLive` now d737d8a5237ad85eadd0e8787493334e0dbb89d7